model/src/main/java/com/ge/predix/acs/rest/AttributeAdapterConnection.java

```diff
@@ -48,6 +54,9 @@ public void setUaaTokenUrl(final String uaaTokenUrl) {
         this.uaaTokenUrl = uaaTokenUrl;
     }
 
+    @ApiModelProperty(
+            value = "An OAuth client that coud be used to obtain an access token for this adapter",
```

Comment:
Typo: coud -> could. Also might be more clearly worded as "OAuth client used to obtain an access token for this adapter"?